spec.html

```diff
@@ -8980,13 +8982,16 @@ <h1>[[GetOwnProperty]] ( _P_ )</h1>
             A property cannot be reported as non-existent, if it exists as a non-configurable own property of the target object.
           </li>
           <li>
-            A property cannot be reported as non-existent, if it exists as an own property of the target object and the target object is not extensible.
+            A property cannot be reported as non-existent, if the target object is not extensible, unless it does not exist as an own property of the target object.
```

Comment:
i've reworded this

Reply:
@claudepache what about:
```suggestion
            A property cannot be reported as non-existent, unless either the target object is extensible, or it does not exist as an own property of the target object.
```